lxd/device/nic_bridged.go

```diff
@@ -99,7 +99,7 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 
 		netConfig := n.Config()
 
-		if d.config["ipv4.address"] != "" {
+		if shared.StringInSlice(d.config["ipv4.address"], []string{"", "none"}) {
```

Comment:
`none` isn't a valid value for a `ipv4.address` when set on a NIC.
It's a valid value when set on a network.